afb-binding/src/binding.rs

```diff
@@ -122,9 +123,9 @@ pub fn binding_init(rootv4: AfbApiV4, jconf: JsoncObj) -> Result<&'static AfbApi
         api.set_permission(AfbPermission::new(to_static_str(value)));
     };
 
-    if let Ok(value) = jconf.get::<i32>("verbosity") {
-        api.set_verbosity(value)?;
-    };
+    // if let Ok(value) = jconf.get::<i32>("verbosity") {
```

Comment:
Why ?

Reply:
I remember that part was causing an error during local build. To bypass it temporarily I commented it out.

Reply:
Is the issue confirmed ?If not, could you remove the comments ?

Reply:
Yes it causes an error (the `?` operator cannot be applied to type `&mut afbv4::apiv4::AfbApi`) I will remove '?'

Reply:
Done

Reply:
:thinking: I can't reproduce the error

Reply:
```
cargo build --features afbv4
   Compiling chmgr-binding v0.0.1
error[E0277]: the `?` operator can only be applied to values that implement `Try`
   --> afb-binding/src/binding.rs:126:9
    |
126 |         api.set_verbosity(value)?;
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^ the `?` operator cannot be applied to type `&mut afbv4::apiv4::AfbApi`
    |
    = help: the trait `Try` is not implemented for `&mut afbv4::apiv4::AfbApi`
```

Reply:
The `--features`  option will use the version of afb-librust set in Cargo.lock
And this version is outdated. It dates back to a version where ` set_verbosity()`  returns `&mut Self`. Now it should return `Result<&mut Self, AfbError>` 

I am not really happy with the fact that we have Cargo.lock tracked by git, but it is apparently needed by our build platform.

So you would need something like `rm Cargo.lock` before the invocation of cargo build